cypress/fixtures/apps.js

```diff
@@ -17,4 +17,5 @@ export const GRAASP_APP_ITEM = {
     isPinned: false,
     showChatbox: false,
   },
+  tags: [],
```

Comment:
I think you can avoid adding tags everywhere if you edit the mock (avoid the mock to throw).